plugins/node/opentelemetry-instrumentation-express/src/types.ts

```diff
@@ -97,4 +107,7 @@ export interface ExpressInstrumentationConfig extends InstrumentationConfig {
   /** Ignore specific layers based on their type */
   ignoreLayersType?: ExpressLayerType[];
   spanNameHook?: SpanNameHook;
+
+  /** Function for adding custom attributes on Express request */
+  requestHook?: ExpressRequestCustomAttributeFunction;
```

Comment:
could you document it on the readme ?

Reply:
Doing it right now; somehow I didn't commit the file. Thanks for noticing!